### PR TITLE
feat(templates): update README and LICENSE for widget templates

### PR DESCRIPTION
**Pull request by eunjae-lee**
## Summary

This PR adds `LICENSE` file to the widget templates and update the default README template files.

### CODE REVIEW

File: src/templates/InstantSearch.js widget/README.md [hunk 1]
```diff
@@ -151,3 +158,47 @@ search.addWidgets([
   }),
 ]);
 ```
+
+## Test
```
Comment (francoischalifour) [inline src/templates/InstantSearch.js widget/README.md:162]: How about we put all the following sections in a "Development" section?

File: src/templates/InstantSearch.js widget/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-present Algolia, Inc.
```
Comment (Haroenv) [inline src/templates/InstantSearch.js widget/LICENSE:3]:
```suggestion
Copyright (c) {{ currentYearSomehow}}-present {{ organisation }}
```

something like that?

File: src/templates/InstantSearch.js widget/README.md (hunk 1 above)
Comment (Haroenv) [reply src/templates/InstantSearch.js widget/README.md:162]: agreed, maybe even better in contributing.md?

File: src/templates/React InstantSearch widget/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-present Algolia, Inc.
```
Comment (Haroenv) [inline src/templates/React InstantSearch widget/LICENSE:3]: same comment

File: src/templates/React InstantSearch widget/README.md [hunk 4]
```diff
@@ -94,3 +108,7 @@ When you are ready, push the git tag and run `npm publish`.
 If you want to publish it as a public scoped package, run `npm publish --access public` the first time.
 
 [To know more about `standard-version`, read this →](https://github.com/conventional-changelog/standard-version#cli-usage)
```
Comment (Haroenv) [inline src/templates/React InstantSearch widget/README.md:110]: same here, we should move these parts to a contributing.md

**Review by Haroenv — COMMENTED** [review]
good idea!

**Comment by eunjae-lee** [comment]
Thanks for the feedback @francoischalifour @Haroenv 
The LICENSE.md.hbs file now applies the current year and the organization name.
The chunk in the README is extracted to CONTRIBUTING.md too.

File: e2e/__snapshots__/templates.test.js.snap [hunk 5]
```diff
@@ -3439,12 +3488,18 @@ instantsearchJsApp({
 or an \`HTMLElement\`:
 
 \`\`\`js
-instantsearchJsApp({
+{
+  {
+    camelCaseName;
```
Comment (Haroenv) [inline e2e/__snapshots__/templates.test.js.snap]: this seems wrong

Comment (eunjae-lee) [reply e2e/__snapshots__/templates.test.js.snap]: It's fixed and not there anymore.

```
{{ camelCaseName }}({
  container: '#{{ name }}',
  // ...
});
```

↑ this snippet in README.md was prettified at that moment, which caused that issue.

File: src/templates/InstantSearch.js widget/LICENSE.md.hbs [hunk 6]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) {{ currentYear }}-present {{ organization }}, Inc.
```
Comment (Haroenv) [inline src/templates/InstantSearch.js widget/LICENSE.md.hbs:3]:
```suggestion
Copyright (c) {{ currentYear }}-present {{ organization }}.
```

Comment (Haroenv) [reply src/templates/InstantSearch.js widget/LICENSE.md.hbs:3]: not everything is Inc, as well as not being a big deal if it's missing

File: src/templates/React InstantSearch widget/LICENSE.md.hbs [hunk 7]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) {{ currentYear }}-present {{ organization }}, Inc.
```
Comment (Haroenv) [inline src/templates/React InstantSearch widget/LICENSE.md.hbs:3]:
```suggestion
Copyright (c) {{ currentYear }}-present {{ organization }}.
```

Comment (eunjae-lee) [reply src/templates/React InstantSearch widget/LICENSE.md.hbs]: nice catch! https://github.com/algolia/create-instantsearch-app/pull/459/commits/562f9bf54510b97fa775bdcf67db179f3a517687

**Review by Haroenv — APPROVED** [review]

**Review by francoischalifour — APPROVED** [review]

**Pull request merged**